docs/docs/en/guides/develop/selective-testing.md

```diff
@@ -1,42 +1,26 @@
 ---
 title: Selective testing
-titleTemplate: :title · Test · Develop · Guides · Tuist
+titleTemplate: :title · Develop · Guides · Tuist
 description: Use selective testing to run only the tests that have changed since the last successful test run.
 ---
 
 # Selective testing {#selective-testing}
 
-> [!IMPORTANT] REQUIREMENTS
-> - A <LocalizedLink href="/guides/develop/projects">Tuist Project</LocalizedLink>
-> - A <LocalizedLink href="/server/introduction/accounts-and-projects">server account and project</LocalizedLink>
-
 As your project grows, so does the amount of your tests. For a long time, running all tests on every PR or push to `main` takes tens of seconds. But this solution does not scale to thousands of tests your team might have.
 
-On every test run on the CI, you probably build a project with cleaned derived data and re-run all the tests, regardless of the changes. `tuist test` helps you to drastically decrease the build time and then running the tests themselves.
-
-## Running tests selectively {#running-tests-selectively}
-
-To run tests selectively, use the `tuist test` command. The command hashes your project the same way it does for <LocalizedLink href="/guides/develop/build/cache#cache-warming">warming the cache</LocalizedLink>, and on success, it persists the hashes on to determine what has changed in future runs.
-
-In future runs `tuist test` transparently uses the hashes to filter down the tests to run only the ones that have changed since the last successful test run.
-
-For example, assuming the following dependency graph:
+On every test run on the CI, you most likely re-run all the tests, regardless of the changes. Tuist's selective testing helps you to drastically speed up running the tests themselves by running only the tests that have changed since the last successful test run.
```

Comment:
I'd add a reference to the hashing page (/en/guides/develop/projects/hashing) if they want to read more about how changes are detected.